advanced-topics/Logging-Loggly/src/PrismSample/App.xaml.cs

```diff
@@ -7,6 +7,7 @@
 using PrismSample.ViewModels;
 using PrismSample.Views;
 using Xamarin.Forms;
+using LogglyOptions = PrismSample.Logging.LogglyOptions;
```

Comment:
remove this

Reply:
Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build. 
This fixes an ambiguous reference between PrismSample.Logging.LogglyOptions and Prism.Logging.Loggly.LogglyOptions that was introduce by Prism.Logging.Loggly exposing a LogglyOptions. I don't really know much about Prism.Logging.Loggly so I don't know why it now exposes a LogglyOptions, but in the readme it says to implement ILogglyOptions and register it (https://github.com/dansiegel/Prism.Plugin.Logging#loggly).

I could rename the sample's LogglyOptions or I could create a new Prism.Logging.Loggly LogglyOptions, configure it, and register it instead. What do you think?

Reply:
Removing this change will cause advanced-topics/Logging-Loggly to no longer be able to build.
This fixes an ambiguous reference between PrismSample.Logging.LogglyOptions and Prism.Logging.Loggly.LogglyOptions that was introduce by Prism.Logging.Loggly exposing a LogglyOptions. I don't really know much about Prism.Logging.Loggly so I don't know why it now exposes a LogglyOptions, but in the readme it says to implement ILogglyOptions and register it (https://github.com/dansiegel/Prism.Plugin.Logging#loggly).

I could rename the sample's LogglyOptions or I could create a new Prism.Logging.Loggly LogglyOptions, configure it, and register it instead. What do you think?

Reply:
oh, sorry, I should have paid more attention. This is not the contoso sample. I though all changes were made to a single  sample. Disregard

Reply:
I removed the sample's LogglyOptions and am now registering a configured instance of the plugin's LogglyOptions. I believe this is how it's intended to be used.

Reply:
Didn't see your comment (it wasn't showing at all) before posting my last comment in the conversation. Sorry about that.
It looks like you merged but didn't get those changes though.